src/main/java/org/tron/core/services/http/JsonFormat.java

```diff
@@ -0,0 +1,1637 @@
+      case GROUP:
+        generator.print("{");
+        print((Message) value, generator);
+        generator.print("}");
+        break;
+    }
+  }
+
+  protected static void printUnknownFields(UnknownFieldSet unknownFields, JsonGenerator generator)
+      throws IOException {
+    boolean firstField = true;
+    for (Map.Entry<Integer, UnknownFieldSet.Field> entry : unknownFields.asMap().entrySet()) {
+      UnknownFieldSet.Field field = entry.getValue();
+
+      if (firstField) {
+        firstField = false;
+      } else {
+        generator.print(", ");
+      }
+
+      generator.print("\"");
+      generator.print(entry.getKey().toString());
+      generator.print("\"");
+      generator.print(": [");
+
+      boolean firstValue = true;
+      for (long value : field.getVarintList()) {
+        if (firstValue) {
+          firstValue = false;
+        } else {
+          generator.print(", ");
+        }
+        generator.print(unsignedToString(value));
+      }
+      for (int value : field.getFixed32List()) {
+        if (firstValue) {
+          firstValue = false;
+        } else {
+          generator.print(", ");
+        }
+        generator.print(String.format((Locale) null, "0x%08x", value));
+      }
+      for (long value : field.getFixed64List()) {
+        if (firstValue) {
+          firstValue = false;
+        } else {
+          generator.print(", ");
+        }
+        generator.print(String.format((Locale) null, "0x%016x", value));
+      }
+      for (ByteString value : field.getLengthDelimitedList()) {
+        if (firstValue) {
+          firstValue = false;
+        } else {
+          generator.print(", ");
+        }
+        generator.print("\"");
+        generator.print(escapeBytes(value));
+        generator.print("\"");
+      }
+      for (UnknownFieldSet value : field.getGroupList()) {
+        if (firstValue) {
+          firstValue = false;
+        } else {
+          generator.print(", ");
+        }
+        generator.print("{");
+        printUnknownFields(value, generator);
+        generator.print("}");
+      }
+      generator.print("]");
+    }
+  }
+
+  /**
+   * Convert an unsigned 32-bit integer to a string.
+   */
+  private static String unsignedToString(int value) {
+    if (value >= 0) {
+      return Integer.toString(value);
+    } else {
+      return Long.toString((value) & 0x00000000FFFFFFFFL);
+    }
+  }
+
+  /**
+   * Convert an unsigned 64-bit integer to a string.
+   */
+  private static String unsignedToString(long value) {
+    if (value >= 0) {
+      return Long.toString(value);
+    } else {
+      // Pull off the most-significant bit so that BigInteger doesn't think
+      // the number is negative, then set it again using setBit().
+      return BigInteger.valueOf(value & 0x7FFFFFFFFFFFFFFFL).setBit(63).toString();
+    }
+  }
+
+  /**
+   * An inner class for writing text to the output stream.
+   */
+  protected static class JsonGenerator {
+
+    Appendable output;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Explicitly declare the visibility for "output". ([squid:S2039](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2039))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1114&issues=AWRjYLgbMt9UOhrw8p6o&open=AWRjYLgbMt9UOhrw8p6o)

[ISSUE_KEY]: AWRjYLgbMt9UOhrw8p6o